docs/get-started/how-to/verify-smart-contract/tenderly.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Tenderly
+description: Tenderly is a comprehensive Web3 development infrastructure that enables developers to build, test, and monitor decentralized applications with support for Virtual TestNets and contract verification using both Hardhat and Foundry frameworks.
```

Comment:
```suggestion
description: Development infrastructure for building, testing, and monitoring dapps.
```

Descriptions need to be shorter in order to fit onto social cards, this is a suggestion, so feel free to modify